test_common/harness/kernelHelpers.cpp

```diff
@@ -686,11 +686,32 @@ static int create_single_kernel_helper_create_program_offline(cl_context context
 
         size_t length = modifiedKernelBuf.size();
         log_info("offlineCompiler: clCreateProgramWithSource replaced with clCreateProgramWithIL\n");
+        if (gCoreILProgram) {
+            *outProgram = clCreateProgramWithIL(context, &modifiedKernelBuf[0], length, &error);
+        }
+        else {
+            cl_platform_id platform;
+            error = clGetDeviceInfo(device, CL_DEVICE_PLATFORM, sizeof(cl_platform_id), &platform, NULL);
+            print_error(error, "clGetDeviceInfo for CL_DEVICE_PLATFORM failed");
```

Comment:
think the result needs checking here as well

Reply:
@gwawiork I think this still needs the error code checked.